laravel/database/seeds/DatabaseSeeder.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+use Illuminate\Database\Seeder;
+
+class DatabaseSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        $this->call(ConfigTableSeeder::class);
```

Comment:
This is where you define which seeders should get called when you seed the database